src/main/java/duke/command/Command.java

```diff
@@ -5,16 +5,34 @@
 import duke.task.TaskList;
 import duke.ui.Ui;
 
+/**
+ * Commands that can be executed by Duke.
+ *
+ * @author Cheong Yee Ming
+ * @version Duke A-JavaDoc
+ */
 public abstract class Command {
     protected final TaskList taskList;
     protected final Storage storage;
     protected final Ui ui;
 
+    /**
+     * Constructor for a DukeCommand.
```

Comment:
You mention a `DukeCommand` in the JavaDocs, but the constructor creates a `Command`, so it might be good to streamline this and standardize to 1 name for the `Command` class